parsers/httpParser.cpp

```diff
@@ -100,6 +101,18 @@ void decapitalizeHeaderFields(std::string& Header) {
 	}
 }
 
+void checkMultipartDelimiter(Request& request) {
+	std::map<std::string,std::string>::iterator it;
+	it = request.HeaderFields.find("content-type");
+	if (it == request.HeaderFields.end()) {
+		return;
+	}
+	if(it->second.find("multipart/form-data;") == std::string::npos &&
+		it->second.find("boundary=") == std::string::npos) {
```

Comment:
should it not be 
`it->second.find("multipart/form-data;") != std::string::npos &&
		it->second.find("boundary=") == std::string::npos`
also a check maybe would be nice if the delimiter is between 1 and 70 characters long